src/app/modules/submission/multirespondent-submission/multirespondent-submission.service.ts

```diff
@@ -801,6 +862,23 @@ export const performMultiRespondentPostSubmissionUpdateActions = ({
     submissionId,
   }
 
+  //send respondent copy emails if needed (before rejection/approved etc.)
+  if (respondentEmails) {
+    sendMrfRespondentCopyEmails({
+      form,
+      responses,
+      submissionId,
+      attachments,
+      respondentEmails,
+    }).mapErr((error) => {
+      logger.error({
+        message: 'Send multirespondent respondent copy email error',
+        meta: logMeta,
+        error,
+      }) // return nothing since if successful submission does not depend on this email submission
```

Comment:
Comment: This is a good example of comment that is used to explain why, though it'll be good at add some punctuation, as it took me a couple reads to understand it.